### PR TITLE
Longevity with multi keyspaces

### PR DESCRIPTION
**Pull request by roydahan**
DO NOT merge yet - require extensive testing

Refactoring test_custom_time to support multi keyspace with round_robin.

### CODE REVIEW

File: longevity_test.py [hunk 1]
```diff
@@ -41,32 +71,64 @@ def test_custom_time(self):
...
-            if 'profile' in stress_cmd:
-                cs_profile = re.search('profile=(.*)yaml', stress_cmd).group(1) + 'yaml'
-                cs_profile = os.path.join(os.path.dirname(__file__), 'data_dir', os.path.basename(cs_profile))
-                params.update({'profile': cs_profile})
-            self.log.debug('stress cmd: {}'.format(stress_cmd))
-            stress_queue.append(self.run_stress_thread(**params))
-            if 'profile' in params:
-                del params['profile']
-        if not prepare_write_cmd:
+
```
Comment (larisau) [inline longevity_test.py]: lines 111-115 moved to _run_all_stress_cmds, so you can remove it here

File: longevity_test.py [hunk 2]
```diff
@@ -15,19 +15,49 @@
...
+    def _parse_stress_cmd(self, stress_cmd, params):
+        # Due to an issue with scylla & cassandra-stress - we need to create the counter table manually
+        if 'counter_' in stress_cmd:
+            self._create_counter_table()
+
+        # When using cassandra-stress with "user profile" the profile yaml should be provided
+        if 'profile' in stress_cmd:
+            cs_profile = re.search('profile=(.*)yaml', stress_cmd).group(1) + 'yaml'
+            cs_profile = os.path.join(os.path.dirname(__file__), 'data_dir', os.path.basename(cs_profile))
+            params.update({'profile': cs_profile})
```
Comment (larisau) [inline longevity_test.py]: return params

File: longevity_test.py [hunk 3]
```diff
@@ -15,19 +15,49 @@
...
+            params.update({'stress_cmd': stress_cmd})
+            self._parse_stress_cmd(stress_cmd, params)
+
+            # Run all stress commands
+            self.log.debug('stress cmd: {}'.format(stress_cmd))
+            stress_queue.append(self.run_stress_thread(**params))
+
+            # Remove "user profile" param for the next command
+            if 'profile' in params:
+                del params['profile']
```
Comment (larisau) [inline longevity_test.py]: return stress_queue

Comment (roydahan) [reply longevity_test.py]: fixed.

File: longevity_test.py (hunk 2 above)
Comment (roydahan) [reply longevity_test.py]: fixed.

**Comment by roydahan** [comment]
Merging after testing:
- Longevity multi keyspaces with write & read
- Longevity multi keyspaces with write & read (without nemesis during the write)
- Regular 1TB Longevity

**Pull request merged**